Assert.AreEqual(5, calc.Add(2, 3), "Adding positives not as expected");\nAssert.AreEqual(-5, calc.Add(-2, -3), "Adding negatives not as expected");\nAssert.AreEqual(-2, calc.Add(-3, 2), "Adding one positive and one negative not as expected");\n\n// your framework might provide a cleaner way of doing this:\ntry {\n  int result = calc.Add(Int32.Max, 5);\n  Assert.Fail("Expected overflow error. Received: " + result);\n} catch(Exception e) {\n  // This should be a more specific error that I'm not looking up\n}